scripts/vaccination_alerts/generate-users-to-email.ts

```diff
@@ -5,10 +5,69 @@ import {
   EmailFips,
   getLocationsToAlert,
   readVaccinationAlerts,
-  getUserLocationsToAlert,
   groupByFips,
+  DEFAULT_ALERTS_FILE_PATH,
+  getStateFipsCodesSet,
 } from './utils';
 
+import * as yargs from 'yargs';
+import { FipsCode } from '../../src/common/regions';
+import { RegionVaccinePhaseInfo } from '../../src/cms-content/vaccines/phases';
+
+const buildEmailsToAlertByFips = (
+  fipsCodesToAlert: FipsCode[],
+  alertSubscriptions: FirebaseFirestore.QuerySnapshot<
+    FirebaseFirestore.DocumentData
+  >,
+) => {
+  const emailFipsList: EmailFips[] = [];
+
+  alertSubscriptions.forEach(emailLocationsDoc => {
+    const email = emailLocationsDoc.id;
+    const { locations } = emailLocationsDoc.data();
+
+    // Currently only alerting on states, so pull state fips codes from subscribed locations.
+    const subscribedStateFips = getStateFipsCodesSet(locations);
+
+    const userFipsToAlert = subscribedStateFips
+      .intersection(fipsCodesToAlert)
+      .value();
```

Comment:
It feels a bit weird for me to break a `_.chain` sequence, consider returning an array from `getStateFipsCodesSet` so we can just use `_.intersection` here.